test/boost/test_config.yaml

```diff
@@ -16,6 +16,7 @@ run_first:
 no_parallel_cases:
     - logalloc_test
     - logalloc_standard_allocator_segment_pool_backend_test
+    - gcp_object_storage_test
```

Comment:
If the fixtuxuture would be external to the process, this wouldn't be necessary (but not so important).